tests/e2e/lib/dpa_helpers.go

```diff
@@ -102,26 +108,32 @@ func (v *DpaCustomResource) Build(backupRestoreType BackupRestoreType) error {
 	for _, plugin := range dpaInstance.Spec.Configuration.Velero.DefaultPlugins {
 		defaultPlugins[plugin] = emptyStruct{}
 	}
-	featureFlags := make(map[string]emptyStruct)
+	veleroFeatureFlags := make(map[string]emptyStruct)
 	for _, flag := range dpaInstance.Spec.Configuration.Velero.FeatureFlags {
-		featureFlags[flag] = emptyStruct{}
+		veleroFeatureFlags[flag] = emptyStruct{}
 	}
+	dpaInstance.Spec.Features = &oadpv1alpha1.Features{DataMover: &oadpv1alpha1.DataMover{Enable: false}}
 	switch backupRestoreType {
 	case RESTIC:
 		dpaInstance.Spec.Configuration.Restic.Enable = pointer.Bool(true)
 		delete(defaultPlugins, oadpv1alpha1.DefaultPluginCSI)
-		delete(featureFlags, "EnableCSI")
+		delete(veleroFeatureFlags, "EnableCSI")
 	case CSI:
 		dpaInstance.Spec.Configuration.Restic.Enable = pointer.Bool(false)
 		defaultPlugins[oadpv1alpha1.DefaultPluginCSI] = emptyStruct{}
-		featureFlags["EnableCSI"] = emptyStruct{}
+		veleroFeatureFlags["EnableCSI"] = emptyStruct{}
+	case CSIDataMover:
+		dpaInstance.Spec.Configuration.Restic.Enable = pointer.Bool(false)
+		defaultPlugins[oadpv1alpha1.DefaultPluginCSI] = emptyStruct{}
+		veleroFeatureFlags["EnableCSI"] = emptyStruct{}
+		dpaInstance.Spec.Features.DataMover.Enable = true
```

Comment:
Do we also need to set `dataMover.credentialName` ?

Reply:
I thought you can just use default credential name? but yes can set it.

Reply:
resolved via 2704f28...a51f67d